slynk/slynk.lisp

```diff
@@ -1625,7 +1630,10 @@ converted to lower case."
                   (process-form-for-emacs (cdr form))))
     (character (format nil "?~C" form))
     (symbol (concatenate 'string (when (eq (symbol-package form)
-                                           #.(find-package "KEYWORD"))
```

Comment:
I think you can get what you want here more easily with just `(find-package '#:keyword)`.  That symbol will be upcased in an ANSI CL, and will stay lower-case in a case-sensitive CL.